BALSAMIC/utils/rule.py

```diff
@@ -288,6 +288,19 @@ def get_delivery_id(
     return delivery_id
 
 
+def get_pon_cnn(config: dict) -> str:
+    """Returns path of pon_cnn for TGA workflow
+
+    Args:
+        config: a config dictionary
+
+    Returns:
+       Returns path of pon_cnn generated by cnvkit
+
+    """
+    return Path(config["panel"]["pon_cnn"]).as_posix()
```

Comment:
As in my preview review: It would be cleaner to move the pon cnn logic from smk to this method, 

```suggestion
    return config["panel"]["pon_cnn"] if "pon_cnn" in config["panel"] else " "
```